transitions/registration.js

```diff
@@ -137,9 +137,8 @@ module.exports = {
      * Given a form code and config array, return config for that form.
      * */
     getRegistrationConfig: function(config, form_code) {
-        var regex = new RegExp('^\W*' + form_code + '\\W*$','i');
```

Comment:
we didn't need to define this twice, there is a util, which also documents why (ish)